test/main/vct/test/integration/examples/TechnicalSpec.scala

```diff
@@ -434,6 +434,8 @@ vercors should verify using silicon in "example using string primitive" pvl
 
   vercors should verify using anyBackend example "technical/labeledEmbeddedStatement.c"
 
+  vercors should verify using anyBackend example "technical/intBoolCoercion.c"
```

Comment:
Does this actually pass? I would not expect it without the changes in CPromoting

Reply:
Yes, it does. Note that `isCPromoting` right now does not have cases for CoerceCIntBool and CoerceBoolCInt, so the default `true` applies. The two `false` cases you mention in your comment above are in `isPromoting`, not in `isCPromoting`.